Example Code/Reflowduino_Demo/Reflowduino_Demo.ino

```diff
@@ -269,7 +400,20 @@ void loop() {
     previousMillis = millis();
     Serial.print("--> Temperature: "); // The right arrow means it's sending data out
     Serial.print(temperature);
-    Serial.println(" *C");
+    Serial.print(" *C; reflow? ");
```

Comment:
The debug could be a good idea but many people won't have it connected to a computer. Maybe it would be better to send these states to the app instead to keep track of the current state.